src/main/java/training/supportbank/transactionhandlers/Formatter.java

```diff
@@ -1,15 +1,17 @@
-package training.supportbank;
+package training.supportbank.transactionhandlers;
+
+import training.supportbank.models.Transaction;
 
 import java.math.BigDecimal;
 import java.util.*;
 
 public class Formatter {
 
     public static String convertToCurrency(BigDecimal value, Locale locale) {
-        String currencyValueString;
-        String outputCurrency = Currency.getInstance(locale).getSymbol(locale);
-        currencyValueString = outputCurrency + value.setScale(2).abs().toString();
-        return currencyValueString;
+        String currencyValue;
+        String localeCurrency = Currency.getInstance(locale).getSymbol(locale);
+        currencyValue = localeCurrency + value.setScale(2).abs().toString();
```

Comment:
Does this need to be on a separate line to the declaration on line 11? (In fact, does this need to be a variable at all? You could just return the result.)